### DISCUSSION TITLE
New Etsy API Bill-Charge Endpoint?

### DISCUSSION THREAD

**Discussion by Miotares** [etsy/open-api#1195] [Ideas]
Hello fellow coders,

I am having a hard time finding the bill charge endpoint in the new Etsy API. Is it already implemented? Thanks!

**Comment by HiveBotNoon** [comment]
Apparently this endpoint is retired with the new V3 API. You can find all retired resources under the following link:

https://developers.etsy.com/documentation/migration/retired-resources/

I am wondering why this endpoint was retired, as it is an important insight. As I am currently working on building a SAAS application it would be cruicial to have this endpoint added to the new V3 API.
